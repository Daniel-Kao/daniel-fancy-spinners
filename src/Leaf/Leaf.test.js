import React from 'react'
import renderer from 'react-test-renderer'
import { Leaf } from '..'

it('renders correctly', () => {
  const tree = renderer.create(<Leaf />).toJSON()
  expect(tree).toMatchSnapshot()
})
