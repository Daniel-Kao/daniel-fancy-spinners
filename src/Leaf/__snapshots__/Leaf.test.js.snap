// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loadingio-spinner-spinner-dpg9d4p7mc"
>
  <div
    className="ldio-f4nowbdfp7q"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
